desktop/apps/artwork/components/partner_stub/components/inquire_via_phone_modal.coffee

```diff
@@ -0,0 +1,29 @@
+AuthModalView = require '../../../../../components/auth_modal/view.coffee'
+{ templateMap } = require '../../../../../components/auth_modal/maps.coffee'
+mediator = require '../../../../../lib/mediator.coffee'
+
+templateMap['phone_number'] = -> require('../templates/phone_number.jade') arguments...
+
+module.exports = class InquireViaPhoneModalView extends AuthModalView
+  template: ->
+    templateToRender = if sd.CURRENT_USER then 'phone_number' else @state.get('mode')
+    templateMap[templateToRender] arguments...
+
+  onSubmitSuccess: (model, response, options) =>
+    analyticsHooks.trigger "auth:#{@state.get 'mode'}"
```

Comment:
This is actually quite tricky, since calling `super` will refresh the entire page while in this case we only need to swap the content. We also need to add a context to the analytics (see conversation on Slack: https://artsy.slack.com/archives/C03JJ2RDG/p1493315865716531) which I'm not sure how to do so. I'm totally open to suggestions!